news/2021-05-20-spring-fanart-contest-results.md

```diff
@@ -0,0 +1,127 @@
+
+Dappled light enchanted **3197** voters in [**McFriedFries**](https://osu.ppy.sh/users/12694139)'s picnic piece sporting pippi, Mocha and Chirou enjoying some time (and burgers) in the sun.
+
+There's just so much going on in this piece that it is hard to praise all of it in only a few short paragraphs. Absolutely stunning, and more than deserving of its place in the finalists.
+
+[![](https://assets.ppy.sh/contests/125/banners/Poring.jpg)](https://assets.ppy.sh/contests/125/winners/Poring.png)
+
+**3107** found themselves lost in [**Poring**](https://osu.ppy.sh/users/14743626)'s urban cityscape vista alongside a uniformed pippi in what we now take to be a still shot of the canonical osu! universe, complete with A Labour of Love billboards flying in the distance.
+
+I'll break from tradition here a little bit to mention that this was one of my personal favourites in the entire contest due to how wonderfully stylized, crisp and clean it is. Everything about it feels so delightfully *deliberate* in a way that is hard to describe but very easy to appreciate.
+
+[![](https://assets.ppy.sh/contests/125/banners/araran.jpg)](https://assets.ppy.sh/contests/125/winners/araran.png)
+
+**2767** voters succumbed to the charms of [**araran**](https://osu.ppy.sh/users/8937198)'s catgirl pippi set to a light-infused backdrop of vibrant sakura trees and clouds.
+
+With the one-two punch of gorgeous background and cute mascot all wrapped up in some serious artistic chops, it's no surprise that araran's first fanart entry ended with a spot in the finalists.
+
+[![](https://assets.ppy.sh/contests/125/banners/Jinairu[Sensei].jpg)](https://assets.ppy.sh/contests/125/winners/Jinairu[Sensei].png)
+
+**2651** voters approved of [**Jinairu[Sensei]**](https://osu.ppy.sh/users/10932731)'s take on all of the official osu! mascots (and Chirou!) lounging around in the springtime grass for a feature shot.
+
+You can sure bet that nobody is going to let Mari live down the fact that she fell asleep during this, especially Mani most of all.
+
+[![](https://assets.ppy.sh/contests/125/banners/Ayamy.jpg)](https://assets.ppy.sh/contests/125/winners/Ayamy.png)
+
+**2635** people stopped to marvel at [**Ayamy**](https://osu.ppy.sh/users/10275467)'s take on a blooming pippi undertaking what appears to be a bread run in the midst of a sunlit, springtime day.
+
+Nothing further will be mentioned, especially not the low-hanging fru--damnit.
+
+[![](https://assets.ppy.sh/contests/125/banners/ShovelKun.jpg)](https://assets.ppy.sh/contests/125/winners/ShovelKun.png)
+
+**2406** voters were taken by the hand to a land that they could understand by [**ShovelKun**](https://osu.ppy.sh/users/9948284)'s most adorable take on Mocha out on an evening springtime walk.
+
+An extra special mention goes out to the creative use of theme here - to my knowledge, nobody else delved into the evening element quite so deeply, and it really produces a stunning result here overall.
```

Comment:
```suggestion
An extra special mention goes out to the creative use of theme here. To my knowledge, nobody else delved into the evening element quite so deeply, and it really produces a stunning result here overall.
```

Reply:
pause and join with dash/hyphen is much more natural here imo

Reply:
disagree; sentence is too long. when reading, there's a longer pause than what a dash would suggest. a full stop gives gravity to the first sentence.